operator/src/main/resources/scripts/startServer.sh

```diff
@@ -137,6 +149,9 @@ function copySitCfg() {
   if [ $? = 0 ]; then
     for local_fname in ${src_dir}/${fil_prefix}*.xml ; do
       copyIfChanged $local_fname $tgt_dir/`basename ${local_fname/${fil_prefix}//}`
+      trace "Printing contents of situational configuration file $local_fname:"
+      file_content=`cat $local_fname`
+      echo "$file_content"
```

Comment:
Can 
```
file_content=`cat $local_fname`
echo "$file_content"``` 
be replaced with 
```
cat $local_fname```?

Reply:
Reduced to one line:
echo `cat $local_fname`